frame/offences/benchmarking/src/lib.rs

```diff
@@ -383,42 +375,6 @@ benchmarks! {
 			+ n // nominators slashed
 		);
 	}
-
-	on_initialize {
```

Comment:
we need to run benchmarks before this is merged

Reply:
Do you want me to create a benchmark test for `on_runtime_upgrade` hook?

Reply:
if on_runtime_upgrade hook cost is not straightforward we can simply use whole block weight.